XcodeServerSDK/Server Entities/DeviceSpecification.swift

```diff
@@ -133,7 +139,7 @@ public class DeviceFilter : XcodeServerEntity {
         public static func architectureFromPlatformType(platformType: DevicePlatform.PlatformType) -> ArchitectureType {
             
             switch platformType {
-            case .iOS, .iOS_Simulator, .watchOS, .watchOS_Simulator, .Unknown:
+            case .iOS, .iOS_Simulator, .watchOS, .watchOS_Simulator, .tvOS, .Unknown:
```

Comment:
Any idea where I can find this cause I've looked through the responses and I can't remember finding anything on `platformType` at all.